code/modules/mob/living/carbon/combat.dm

```diff
@@ -48,7 +48,9 @@
 			playsound(loc, I.miss_sound, 50)
 		on_dodge(user, I)
 		return FALSE
-	if(I.hitsound)
+	if(istype(I,/obj/item/tool/crowbar/red))
+		playsound(loc, "crowbar_hitbod", 50, 1, -1, frequency = 22000) // Different sound for living things
```

Comment:
![image](https://i.imgur.com/1CBtKuV.gif)

Reply:
I think it'd be better if I just made them normal frequency files instead of doing this, it's some dumb workaround..